apps/heft/src/pluginFramework/logging/LoggingManager.ts

```diff
@@ -47,25 +48,40 @@ export class LoggingManager {
     }
   }
 
-  public getErrorStrings(): string[] {
+  public getErrorStrings(fileErrorFormat?: FileErrorFormat): string[] {
     const result: string[] = [];
 
     for (const scopedLogger of this._scopedLoggers.values()) {
-      result.push(...scopedLogger.errors.map((error) => `[${scopedLogger.loggerName}] ${error.message}`));
+      result.push(
+        ...scopedLogger.errors.map(
+          (error) => `[${scopedLogger.loggerName}] ${LoggingManager.getErrorMessage(error, fileErrorFormat)}`
+        )
+      );
     }
 
     return result;
   }
 
-  public getWarningStrings(): string[] {
+  public getWarningStrings(fileErrorFormat?: FileErrorFormat): string[] {
     const result: string[] = [];
 
     for (const scopedLogger of this._scopedLoggers.values()) {
       result.push(
-        ...scopedLogger.warnings.map((warning) => `[${scopedLogger.loggerName}] ${warning.message}`)
+        ...scopedLogger.warnings.map(
+          (warning) =>
+            `[${scopedLogger.loggerName}] ${LoggingManager.getErrorMessage(warning, fileErrorFormat)}`
+        )
       );
     }
 
     return result;
   }
+
+  public static getErrorMessage(error: Error, fileErrorFormat?: FileErrorFormat): string {
+    if (error instanceof FileError) {
```

Comment:
Lately we've had issues with `instanceof` not working reliably across packages. Take a look at https://github.com/microsoft/rushstack/pull/2092 for example. 

I haven't had time to investigate it yet, but I wonder if we need to cook up a different approach for this problem, something like how DirectX identifies interface implementors based on an interface GUID.  (?)